crypto/dkg_test.go

```diff
@@ -181,13 +189,17 @@ func dkgCommonTest(t *testing.T, dkg int, n int, threshold int, test testCase) {
 	mrand.Seed(rand)
 	t.Logf("math rand seed is %d", rand)
 	var r1, r2 int
```

Comment:
could you add a comment here explaining what these are?

It seems like they have different meanings depending on the test case -- maybe we could create them in the specific case they are for and give a more descriptive name specific to the testcase?

Reply:
`r1` and `r2` is the number of malicious nodes. `r1` will behave maliciously and will be disqualified. `r2` will behave maliciously at first but will correct their behaviour later and won't be disqualified. 
I'll add more comments.